pkg/proxy/ipvs/proxier.go

```diff
@@ -693,15 +693,6 @@ func (em proxyEndpointsMap) unmerge(other proxyEndpointsMap) {
 // return an error if it fails to get the kernel modules information without error, in which
 // case it will also return false.
 func CanUseIPVSProxier(ipsetver IPSetVersioner) (bool, error) {
-	// Try to load IPVS required kernel modules using modprobe
```

Comment:
Why delete these lines? kube-proxy is not always running inside container. If kube-proxy running on host, it can modprobe kernel modules. Besides, if kube-proxy is running inside container and failed to load kernel modules, it just log warning messages - please double check it.

Reply:
I figured out if kube-proxy run inside the container, it will certainly fail.
But maybe we can accept an warning message. I will get it back.

Reply:
Yeah, that's why we just log warning message instead of existing with error.